index.bs

```diff
@@ -2760,6 +2784,8 @@ To <dfn>trigger event-level attribution</dfn> given an [=attribution trigger=] |
     return it.
 1. Let |report| be the result of running [=obtain an event-level report=] with |sourceToAttribute|, |trigger|,
     and |matchedConfig|.
+1. If |report| is null, return the [=triggering result=]
+    ("<code>[=triggering status/dropped=]</code>", null).
```

Comment:
I think it would be more consistent to send a verbose debug report here, since it is similar to unmatched filters. Thoughts?

Reply:
Agreed, this seems simplest to me.

Reply:
Should there be any new fields in the debug report body? If we use the precedent of mismatched filters, then the answer is no, but it could be helpful to include the trigger's `trigger_data` and the source's set of allowed `trigger_data`. Thoughts?